docs/reference/errors-and-warnings/NU5048.md

```diff
@@ -0,0 +1,63 @@
+---
+title: NuGet Error NU5048
+description: NU5048 Error code
+author: dominoFire
+ms.author: feaguila
+ms.date: 9/4/2019
+ms.topic: reference
+ms.reviewer: karann
+f1_keywords: 
+  - NU5048
+---
+
+# NuGet Error NU5048
+
+<pre>The 'PackageIconUrl'/'iconUrl' element is deprecated. Consider using the 'PackageIcon'/'icon' element instead. Learn more at https://aka.ms/deprecateIconUrl</pre>
+
+
+# Issue
+
+Icon URL is deprecated in favor of embedding the icon inside the NuGet package. Possible causes are:
+
+- When creating a package from a nuspec file, it contains a `<iconUrl/>` entry.
+- When creating a package from a MSBuild project file, it contains a `<PackageIconUrl>` property.
+
+
+# Solution
+
+To stop seeing this warning, add an embedded icon to your package.
+
+For nuspec files, add an `<icon/>` entry that points to the file that will be the package icon:
```

Comment:
Like my comment in the other file, consider whether the elements in the examples below that are not specific to icons add or distract from the example.

Reply:
fixed.